hub/codec/tests/test_codecs.py

```diff
@@ -0,0 +1,92 @@
+import pytest
+import numpy as np
+
+from hub.codec import PngCodec, NumPy, Lz4, Zstd, JpegCodec
+
+
+@pytest.mark.parametrize("from_config", [False, True])
+def test_png_codec(from_config: bool) -> None:
+    codec = PngCodec()
+    if from_config:
+        config = codec.get_config()
+        codec = PngCodec.from_config(config)
+    arr = np.ones((1000, 1000, 3), dtype="uint8")
+    bytes_ = codec.encode(arr)
+    arr_ = codec.decode(bytes_)
+    assert (arr == arr_).all()
```

Comment:
Do you need the `.all()`? Pytest overloads the `assert` operator for iterables; does that apply here?

Reply:
Tried using without all, it gives: `ValueError: The truth value of an array with more than one element is ambiguous. Use a.any() or a.all()`

Reply:
Gotcha. Must be numpy arrays being finnicky